app/models/manageiq/providers/base_manager/refresh_helper_methods.rb

```diff
@@ -0,0 +1,19 @@
+module ManageIQ::Providers::BaseManager::RefreshHelperMethods
+  extend ActiveSupport::Concern
+
+  include Vmdb::Logging
+
+  def memory_logger(ems, message)
+    log_header = format_ems_for_logging(ems)
+    _log.debug("#{log_header} -- #{message} -- Memory usage: #{'%.02f' % collector_memory_usage} MiB")
+  end
+
+  def collector_memory_usage
+    require 'miq-process'
+    MiqProcess.processInfo[:proportional_set_size].to_f / 1.megabyte
+  end
+
+  def format_ems_for_logging(ems)
```

Comment:
If you put the memory_logger in `ManageIQ::Providers::BaseManager::Refresher` you wouldn't need to redefine this

Reply:
@agrare Ah ok :)